UI/DevScreens.cpp

```diff
@@ -1405,6 +1406,7 @@ void FrameDumpTestScreen::CreateViews() {
 		c->SetTag(url);
 		c->OnClick.Handle<FrameDumpTestScreen>(this, &FrameDumpTestScreen::OnLoadDump);
 	}
+	delete dumpsScroll;
```

Comment:
this is wrong. this will be deleted later, it gets added to the view hiearchy so it's being kept track of.

Or, hm, turns out it doesn't - that's a bug. We don't get a scrollview at all.

So instead, above, change the last parameter of "tabHolder->addTab("Dumps", dumps)" to dumpsScroll.

Reply:
> Or, hm, turns out it doesn't - that's a bug. We don't get a scrollview at all.
> 
> So instead, above, change the last parameter of "tabHolder->addTab("Dumps", dumps)" to dumpsScroll.

you have eye eagle, you noticed other mistakes through my PR.

Reply:
fix it https://github.com/hrydgard/ppsspp/pull/18967/commits/9c3c23daa6d3a70921f3aeb3560c81b497065eb4

Reply:
This line still needs to be removed - as now the object is properly in the view hiearchy and will be deleted later.